packages/gestalt/src/Flyout.test.js

```diff
@@ -6,7 +6,11 @@ import Flyout from './Flyout.js';
 it('Flyout renders', () => {
   const wrapper = shallow(
     <Flyout
-      anchor={<button onClick={() => null}> test </button>}
+      anchor={
+        <button onClick={() => null} type="submit">
```

Comment:
@pfarejowicz we should probably change this to `type="button"`. A flyout inside of a form should not be submitting the form. Also `button` is the default for `<Button>`

Reply:
Will change